### PR TITLE
Markdown

### PR DESCRIPTION
**Pull request by Spazcool**
Add markdown support for howto, details and troubleshooting cells from spreadsheet. #50 

markdownsup

### CODE REVIEW

**Comment by lenzai** [comment]
which equipment should we use as test for markdown ?

**Comment by Spazcool** [comment]
@lenzai I've created a test row, row 84, and removed my edits to the real item rows above. As for you other suggestions, I gather OOP means object oriented programming, but I don't understand what you mean.

File: googleSpreadsheet.js [hunk 1]
```diff
@@ -33,7 +34,28 @@ function searchDatabase(query, rows) {
   return matches;
 }
 
+function addSimilarItems(object, objects) {
+  const obj = object;
```
[the remark this answers is not in the document]
Comment (Spazcool) [reply googleSpreadsheet.js]: @lenzai  Again, this is comes from the linter. I didn't want to give you any more error messages, so I added this quick fix. Do you want more errors or is there another solution?

File: views/notFound.ejs [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 <img class="col-xs-12 col-lg-4 col-lg-offset-4" src="/emoji.png" alt="" />
 <article style='margin-top:2em'class="col-xs-12 col-lg-4 col-lg-offset-4">
-  <p>Sorry, item not found with the id:</p>
+  <p>Sorry, item not found with the id: </p>
```
[the remark this answers is not in the document]
Comment (Spazcool) [reply views/notFound.ejs]: I think this was a left over from me hacking my way to a solution, using this page to test as it required the least amount of fiddling to get working. 

I didn't mean to. Sorry.

**Pull request merged**